tests/functions/test_nonstandard_charac.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
```

Comment:
It appears that the tests here are for the function `clean_names`. Could you move them into the `test_clean_names.py` file?

Reply:
**Action item:** As per comment above.